File: hs/53.hs
-- 4075
import Euler(nChooseK)

vv = 1000000
nn = 100

numCombs v n = length $ filter (>v) [nChooseK r k | k <- [1..n], r <- [k..n]]

main = putStrLn $ show $ numCombs vv nn

File: py/53.py
# 4075
import euler

N = 1000000
M = 100

s = 0
f = [1]
f += [euler.product(xrange(1,n+1)) for n in xrange(1, M+1)]
for n in xrange(1,M+1):
    for r in xrange(n+1):
        if f[n] / f[r] / f[n-r] > N:
            s += 1
print s
